pypyr/steps/shell.py

```diff
@@ -33,7 +44,12 @@ def run_step(context):
     # spaces to allow for natural shell language input string.
     logger.debug(f"Executing command string: {context['cmd']}")
 
+    interpolated_string = pypyr.format.string.get_interpolated_string(
+        input_string=context['cmd'],
+        context=context)
+    logger.debug(f"Interpolated string: {interpolated_string}")
```

Comment:
good catch. I'll do you one better and get rid of the second "executing interpolated" altogether - the core shouldn't be writing potentially sensitive context values to the log unless user explicitly told it it to.